cmd/cluster/create.go

```diff
@@ -40,6 +41,7 @@ type Options struct {
 	PublicZoneID       string
 	PrivateZoneID      string
 	Annotations        []string
+	NetworkType        string
```

Comment:
can't this be `hyperv1.NetworkType` so no need to convert later?

Reply:
The command line flag doesn't support enum values from what I can tell. That's why I do the string to enum conversion

Reply:
Specifically `cmd.Flags()` if it did you are totally right though.